pyproject.toml

```diff
@@ -96,6 +96,7 @@ openlm = {version = "^0.0.5", optional = true}
 azure-ai-formrecognizer = {version = "^3.2.1", optional = true}
 azure-ai-vision = {version = "^0.11.1b1", optional = true}
 azure-cognitiveservices-speech = {version = "^1.28.0", optional = true}
+azure-search-documents = {version = "11.4.0a20230509004", source = "azure-sdk-dev", optional = true}
```

Comment:
is this an internal alpha release? don't see it on pypi and some of the imports don't work using latest/beta versions of package

Reply:
@dev2049 this is the source for the package, not currently available on pypi. 

[[tool.poetry.source]]
name = "azure-sdk-dev"
url = "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-python/pypi/simple/"
secondary = true

Reply:
do you now (/are you allowed to say) when there will be an official release?

Reply:
sorry, cannot comment on this. @farzad528 @pablocastro comments here?

Reply:
By June 29th the SDK feed will be public

Reply:
Hi @dev2049 is this a blocker or can we deal with it as a temporary solution? @farzad528

Reply:
@dev2049 would it be blocked until the SDK is publicly available or what if it can be handled with http clients internally would that suffice? Just checking.